deploy/metrics-server-deployment.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: metrics-server
+  namespace: kube-system
```

Comment:
so... as an operator, I kind of like having all the k8s services in one namespace. These comments seem to conflate the default service account in kube-system with the namespace itself? Can't we just make a service account for metrics-server and use that in the kube-system namespace instead?